src/main/java/strategy/ExecuteFruitOperationImpl.java

```diff
@@ -0,0 +1,21 @@
+package strategy;
+
+import dao.Storage;
+import model.FruitTransaction;
+
+public class ExecuteFruitOperationImpl implements ExecuteFruitOperation {
```

Comment:
what handler is it? better create a separate hanlder for each operation, now you have balance and supply. Supply can function as return. So this is a purchase handler?